app/models/tag.rb

```diff
@@ -1123,25 +1120,40 @@ def before_destroy
     if Tag::USER_DEFINED.include?(tag.type) && tag.canonical
       tag.remove_from_autocomplete
     end
-    update_tag_nominations(tag, true)
   end
 
   private
 
-  def update_tag_nominations(tag, deleted=false)
-    values = {}
-    if deleted
-      values[:canonical] = false
-      values[:exists] = false
-      values[:parented] = false
-      values[:synonym] = nil
-    else
-      values[:canonical] = tag.canonical
-      values[:synonym] = tag.merger.nil? ? nil : tag.merger.name
-      values[:parented] = tag.parents.any? {|p| p.is_a?(Fandom)}
-      values[:exists] = true
-    end
-    TagNomination.where(tagname: tag.name).update_all(values)
+  after_save :update_tag_nominations
+  def update_tag_nominations
+    TagNomination.where(tagname: name).update_all(
+      canonical: canonical,
+      synonym: merger.nil? ? nil : merger.name,
+      parented: false, # we'll fix this later
```

Comment:
This threw me for a sec -- I was like, "Well, why isn't this a TODO referencing an issue?"

Could you change it to something like, "We'll fix this later in the callback" instead?